src/main/java/com/xebialabs/restito/semantics/Action.java

```diff
@@ -293,6 +295,33 @@ public Response apply(Response input) {
     }
 
     /**
+     * Creates a sequence action which contains all passed actions and
+     * executes one by one of them in the same order if {@link Action#apply(Response)} is repeated.
+     * If all passed actions has been already applied it behaves like {@link #noop()} action.
```

Comment:
I think, sequence can be complex enough to have its own fluent API. This will also keep the door open for other configs. 

```
sequence(a1, a2).withDefault(defaultAction);
```

It shouldn't be necessary to define `withDefault()` though. `then(sequence())` should still work with noop as default.